Packs/Threat_Crowd/Integrations/ThreatCrowd_v2/ThreatCrowd_v2.py

```diff
@@ -0,0 +1,301 @@
+        votes = json_res.get('votes')
+        if votes == -1:
+            return Common.DBotScore.BAD, 'BAD'
+        elif votes == 0:
+            return Common.DBotScore.SUSPICIOUS, 'SUSPICIOUS'
+        elif votes == 1:
+            return Common.DBotScore.GOOD, 'GOOD'
+
+    return Common.DBotScore.NONE, 'None'
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def ip_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    command_results: List[CommandResults] = []
+    api_url = 'ip/report/'
+    res_limit = None
+    ips = argToList(args.get('ip'))
+    for ip in ips:
+        res = client._http_request(method='GET', url_suffix=api_url, params={'ip': ip})
+        res['value'] = ip
+        score, score_str = _get_dbot_score(res)
+        dbot = Common.DBotScore(
+            ip, DBotScoreType.IP, VENDOR, score, reliability=client.reliability)
+        ip_object = Common.IP(ip, dbot)
+
+        markdown = f"### Threat crowd report for ip {ip}: \n"
+        markdown += f'DBotScore: {score_str} \n'
+
+        if not client.extended_data:
+            res_limit = DEFAULT_RESOLUTION_LIMIT
+        hashes = res.get('hashes')[:res_limit]
+        resolutions = handle_resolutions(res.get('resolutions', []), res_limit)
+        markdown += tableToMarkdown('Resolutions', resolutions)
+        markdown += f"Hashes: \n {hashes} \n"
+        markdown += tableToMarkdown('References', res.get('references'))
+        outputs = {
+            'hashes': hashes,
+            'permalink': res.get('permalink'),
+            'resolutions': resolutions,
+            'references': res.get('references'),
+            'response_code': res.get('response_code'),
+            'votes': res.get('votes'),
+            'value': res.get('value')
+        }
+        command_results.append(CommandResults(
+            outputs_prefix='ThreatCrowd.IP',
+            outputs=outputs,
+            outputs_key_field='value',
+            indicator=ip_object,
+            readable_output=markdown,
+            raw_response=res
+
+        ))
+
+    return command_results
+
+
+def email_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    command_results: List[CommandResults] = []
+    api_url = 'email/report/'
+    emails = argToList(args.get('email'))
+    for email in emails:
+        res = client._http_request(method='GET', url_suffix=api_url, params={'email': email})
+        res['value'] = email
+        score, score_str = _get_dbot_score(res)
+        dbot = Common.DBotScore(
+            email, DBotScoreType.EMAIL, VENDOR, score, reliability=client.reliability)
+        email_object = Common.EMAIL(email, dbot)
+
+        markdown = f"Threat crowd report for Email {email} \n"
+        markdown += f'DBotScore: {score_str} \n'
+        markdown += tableToMarkdown("Results", res)
+
+        command_results.append(CommandResults(
+            outputs_prefix='ThreatCrowd.Account',
+            outputs=res.copy(),
```

Comment:
why is the .copy() needed?

Reply:
res.copy() is being used as it passed by ref, so changing last file's value will also change all previous file's values and so on.
Adding a comment with it.